frame/contracts/src/exec.rs

```diff
@@ -140,19 +165,43 @@ pub trait Ext: sealing::Sealed {
 	///
 	/// Returns `None` if the `key` wasn't previously set by `set_storage` or
 	/// was deleted.
-	fn get_storage(&mut self, key: &StorageKey) -> Option<Vec<u8>>;
+	fn get_storage(&mut self, key: FixSizedKey) -> Option<Vec<u8>>;
```

Comment:
Let's keep the `key` a reference. There is no reason to pass it by value,  right? Same for the other functions.